pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -53,12 +54,16 @@ stdenv.mkDerivation rec {
       pygobject3
       dbus-python
       python-dbusmock
+      pylint
+      argparse-manpage
+      shtab
     ]))
   ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
     mesonEmulatorHook
   ];
 
   buildInputs = [
+    bash-completion
```

Comment:
Looks like zsh completion path needs to be explicitly provided: https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/compare/0.20...0.21#0cc1139e3347f573ae1feee5b73dbc8a8a21fcfa_68_82